### PR TITLE
Adds wagtail integration

### LINKED ISSUE #34: Add wagtail support

### PR DESCRIPTION
**Pull request by fatboystring**
Sorry for the large PR

Closes #34

### CODE REVIEW

File: omniforms/models.py [hunk 1]
```diff
@@ -23,6 +23,33 @@
 import re
 
 
+class OmniFieldQuerySet(models.QuerySet):
+    """
+    Custom QuerySet class for the OmniField model
+    """
+    def field_models(self):
```
Comment (dammitjim) [inline omniforms/models.py]: is it worth refactoring this into the function that takes `model_class` as an arg? It looks like this code is repeated in OmniFormHandlerQuerySet.

Not really necessary if these are the only instances though.

File: omniforms/wagtail/forms.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        :param commit: Whether or not to commit the changes to the database
+        :return: The saved group instance
+        """
+        # We go back to the object to read (in order to reapply) the
+        # permissions which were set on this group, but which are not
+        # accessible in the wagtail admin interface, as otherwise these would
+        # be clobbered by this form.
+        try:
+            untouchable_permissions = self.instance.permissions.exclude(pk__in=self.fields['permissions'].queryset)
+            bool(untouchable_permissions)  # force this to be evaluated, as it's about to change
```
Comment (dammitjim) [inline omniforms/wagtail/forms.py]: Potentially cast to a list, add to comment the reasoning (the queryset is lazy).

File: omniforms/models.py (hunk 1 above)
Comment (fatboystring) [reply omniforms/models.py]: Done in https://github.com/omni-digital/omni-forms/pull/36/commits/b7166c67da6f2b373634cc19eb0ffd1a9c91fc53

File: omniforms/wagtail/forms.py (hunk 2 above)
Comment (fatboystring) [reply omniforms/wagtail/forms.py]: Done in https://github.com/omni-digital/omni-forms/pull/36/commits/616478d79f0834b625d8ab28e7b0e1b3d85b6ee3

File: omniforms/wagtail/forms.py [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+class OmniPermissionFormBase(forms.ModelForm):
+    """
+    Custom form class for rendering omniform permissions in the wagtail admin
+    """
+    class Meta(object):
+        """
+        Django properties
+        """
+        model = Group
+        fields = ("permissions",)
```
Comment (SemVerTsar) [inline omniforms/wagtail/forms.py]: Is it worth changing the `"` to `'` here?

Comment (fatboystring) [reply omniforms/wagtail/forms.py]: Done

**Pull request merged**